packages/typespec-test/test/multiclient_modular/generated/typespec-ts/src/api/bar/operations.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Client } from "../../rest/bar/index.js";
+import { StreamableMethod } from "@azure-rest/core-client";
+import { Resource } from "./models.js";
+import { RequestOptions } from "../../common/interfaces.js";
+import {
+  GetBinary200Response,
+  GetArray200Response,
+  CreateWithHeaders201Response,
+  DeleteWithHeaders204Response,
+} from "../../rest/bar/responses.js";
+
+export interface GetBinaryOptions extends RequestOptions {}
+
+export function _getBinarySend(
+  context: Client.BarContext,
+  options: GetBinaryOptions = { requestOptions: {} }
+): StreamableMethod<GetBinary200Response> {
+  return context
+    .path("/cadl-bar/get-binary")
+    .get({
+      allowInsecureConnection: options.requestOptions?.allowInsecureConnection,
+      skipUrlEncoding: options.requestOptions?.skipUrlEncoding,
+    });
+}
+
+export async function _getBinaryDeserialize(
+  result: GetBinary200Response
+): Promise<any> {
+  return result.body;
+}
+
+/** */
+export async function getBinary(
+  context: Client.BarContext,
+  options: GetBinaryOptions = { requestOptions: {} }
+): Promise<any> {
```

Comment:
Why the return type is `any`?

Reply:
Because the TypeSpec is defined like this 
```
  @route("/get-binary")
  @get
  getBinary(): OkResponse & {
    @header("content-type") contentType: "image/png";
    @body body: bytes;
  };
```
And previously, we had some issues in handling the Anonymous model,  Jose has this temporary workaround for this https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/src/modelUtils.ts#L159-L165